lib/config/argv.c

```diff
@@ -88,9 +88,14 @@ argv_deserialize_to_params(
             free(buf);
 
             if (!node) {
-                CLOG_ERR("Failed to parse %s %s: %s", params_logging_context(params), ps->ps_name, param);
-                err = merr(EINVAL);
-                goto out;
+                if (cJSON_GetErrorPtr()) {
+                    CLOG_ERR("Failed to parse %s %s: %s", params_logging_context(params), ps->ps_name, param);
```

Comment:
I didn't add the ErrorPtr string to the log message here because I felt like since the name was already there, it would be obvious, but I am happy to add it in if others think it would be better.